src/plugins/core/conditional_format.ts

```diff
@@ -389,6 +391,18 @@ export class ConditionalFormatPlugin
     return cmd.ranges.length ? CommandResult.Success : CommandResult.EmptyRange;
   }
 
+  private checkValidRange(cmd: AddConditionalFormatCommand): CommandResult {
+    const ranges = cmd.ranges.map((range) => this.getters.getRangeFromRangeData(range));
+    if (ranges.some((range) => range.sheetId != cmd.sheetId)) {
+      return CommandResult.TargetOutOfSheet;
+    }
+    const stringRanges = ranges.map((range) => this.getters.getRangeString(range, cmd.sheetId));
+    if (stringRanges.some((xc) => !this.getters.isRangeValid(xc))) {
+      return CommandResult.InvalidRange;
+    }
```

Comment:
probably wrong way to do that/useless. We're coming from a RangeData, so either it's a normal rangeData and the XC will always be valid, or it's really malformed and the conversion to range/string will throw and we'll never return `CommandResult.InvalidRange`.

And we already test for invalid zones inside the sheet core plugin.

Reply:
Oh you copied what was already in the data validation. Well either i'm wrong and there is cases where you return `CommandResult.InvalidRange` here, or you can remove it from both.

Reply:
I didn't look at it in details but your comment makes sense.